stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -497,44 +497,40 @@ extension TFState {
       targetNodeSpecs.append(helperFuncNode)
     }
 
-    if returnValues.count > 0 {
-      if _RuntimeConfig.executionMode.isTPU {
-        debugLog("Enable TPU execution.")
-        // When infeed is enabled, run it along with the output tensor nodes
-        // below.
-        let infeedEnqueueNode = TF_GraphOperationByName(graph,
-                                                        "InfeedEnqueueTuple")
-        if let infeedEnqueueNode = infeedEnqueueNode {
-          targetNodeSpecs.append(infeedEnqueueNode)
-          debugLog("Running enqueue with \(inputTensors.count) input tensors.")
-        }
+    if returnValues.count == 0 {
```

Comment:
You can use `if returnValues.isEmpty` instead.

Reply:
Yep, that would be more idiomatic.

Reply:
Done.